meta/SchemaBuilder.php

```diff
@@ -199,6 +202,26 @@ protected function updateColumns() {
         return true;
     }
 
+    private function migrateSingleToMulti($table, $colref) {
+        $sqlSelect = "SELECT pid, rev, col$colref AS value FROM data_$table WHERE latest = 1";
+        $res = $this->sqlite->query($sqlSelect);
+        $valueSet = $this->sqlite->res2arr($res);
+        $this->sqlite->res_close($res);
+        $valueString = array();
+        foreach ($valueSet as $values) {
```

Comment:
Ah okay. I think I got confused by the join later down which I thought is joining multiple values into a single column. But it's not. See my comment there then ;-)